src/Compilers/CSharp/Portable/Binder/Binder_Lambda.cs

```diff
@@ -255,6 +259,13 @@ static void checkAttributes(AnonymousFunctionExpressionSyntax syntax, SyntaxList
             {
                 var hasTypes = parameterSyntaxList[0].Type != null;
 
+                // Implicitly Typed Default Parameters are not allowed
```

Comment:
>Implicitly Typed Default Parameters [](http://example.com/codeflow?start=19&length=35)

Consider using lowercase within the sentence, here and in the matching comment below: "Implicitly typed default parameters ..."